data_dictionary/data_dictionary.cc

```diff
@@ -390,6 +390,12 @@ struct fmt::formatter<data_dictionary::user_types_metadata> {
 };
 
 auto fmt::formatter<data_dictionary::keyspace_metadata>::format(const data_dictionary::keyspace_metadata& m, fmt::format_context& ctx) const -> decltype(ctx.out()) {
-    return fmt::format_to(ctx.out(), "KSMetaData{{name={}, strategyClass={}, strategyOptions={}, cfMetaData={}, durable_writes={}, userTypes={}}}",
-            m.name(), m.strategy_name(), m.strategy_options(), m.cf_meta_data(), m.durable_writes(), m.user_types());
+    fmt::format_to(ctx.out(), "KSMetaData{{name={}, strategyClass={}, strategyOptions={}, cfMetaData={}, durable_writes={}, tablets=",
+            m.name(), m.strategy_name(), m.strategy_options(), m.cf_meta_data(), m.durable_writes());
+    if (m.initial_tablets()) {
```

Comment:
Won't this result in false positive if `tablets = {enabled: true, initial: 0}`?
Best to check replication instead.

Reply:
`init_tablets` is a `std::optional<unsigned>`.
I can change the expression to `initial_tablets.has_value()` to make it clearer

Reply:
> Best to check replication instead.

How would replication help?

Reply:
I meant to check if keyspace has tablets replication strategy. But if `initial` is an optional, then this will already be correct. I don't think `has_value()` is needed, once somebody checks the type of `initial_tablets()`, it will be clear.

Reply:
> I meant to check if keyspace has tablets replication strategy.

There is no such thing today.  The replication strategy is NetworkTopologyStrategy, while tablets have their own options.  From the syntax perspective the tablets options are: `enabled:` (bool) and `initial` (unsigned integer).
In keyspace_metadata we keep only a `std::optional<initial_tablets>` that encapsulates both.